string change_difficulty
---
bool success